lesson02/part1-homework/TagHelpers/InputTagHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace TagHelpers
+{
+    class InputTagHelper : TagHelper
```

Comment:
So this tag helper uses convention in order match what elements it targets. So this works for now, but what happens when you need to write another tag helper that also targets the `<intput />` tag?